src/components/eRACommons.js

```diff
@@ -1,10 +1,10 @@
-import _ from 'lodash';
+import {_, isNil} from 'lodash';
```

Comment:
This is causing some compilation problems in the page. I changed this to 
```
import {get, isNil, isUndefined, merge } from 'lodash';
```
and removed all of the `_.` references which allows the page to render.